<?xml version="1.0"?>

<launch>
    <arg name="veh" default="wamv2" />
    <arg name="ws" default="1" />"

    
    <group if="$(eval veh == 'wamv')">
        <node pkg="vrx_gazebo" type="real2sim_obstacle_circle.py" name="real2sim_obstacle" output="screen">
            <remap from="raw_obstacles" to="/$(arg veh)/raw_obstacles"/>
            <param name="~ws" value="$(arg ws)" />
        </node>
    </group>
    
    <group if="$(eval veh == 'wamv2')">
        <node pkg="vrx_gazebo" type="real2sim_obstacle_line.py" name="real2sim_obstacle" output="screen">
            <remap from="raw_obstacles" to="/$(arg veh)/raw_obstacles"/>
            <param name="~ws" value="$(arg ws)" />
        </node>
    </group>

</launch>
